common/xml/convert_and_merge_composable_tests/explicit-port.golden.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<xml>
+  <port-value-test o="port-name"/>
```

Comment:
Write attribute name?

Reply:
I didn't want to use an existing attribute name...